### ISSUE TITLE
Re-Seeding Database Issue

### ISSUE THREAD

**Issue by anastasiaalt** [anastasiaalt/PatzoParty#5]
After making the changes to the seeds file, I wanted to reseed the database in terminal by . I am getting ​[11:41] 
Error: incomplete SQL: require_relative "config" when I attempt this, which is odd to me because I have require_relative "config" at the top of my seeds.rb file and in my config.rb file I have   :database =>  'db/themeparties.sqlite3'

Any thoughts/ideas/suggestions?

**Comment by phlco** [comment]
You're confusing running SQL files and running RUBY files.  Make sure you're using the right command to run the right type of file.

**Comment by anastasiaalt** [comment]
My understanding was that $ sqlite3 database.sqlite3 < seeds.rb was used to seed the database. I also tried $ ruby database.sqlite3 < seeds.rb and no luck either. :(

**Comment by anastasiaalt** [comment]
Below is the details of an hour's long odyssey down the rabbit hole of re-seeding. Oy vey!

I tried $ ruby seeds.rb and am getting 

$ ruby seeds.rb
seeds.rb:3:in `<main>': uninitialized constant Author (NameError)
anastasiaalt ~/ThemeParties/db on master[!]
$ 

so I studied some differences with old assignments and classwork and noticed this difference in the models for the Cupid assignment:

class_name: "Visit"

I don't really follow what it does, but I tried this in my author model:

has_many :articles, class_name: "Article"

and followed this procedure for all the models accordingly. But no such luck. :(

At Tim's suggestion, I tried adding this to the stop of my seeds file

require "./models/author"
require "./models/article"
require "./models/category"
require "./models/editor"

and got this error:

configuration /Users/anastasiaalt/ThemeParties/db/config.ru not found

so then I changed my config.ru to:

require "db/config"

and got this error:

$ ruby seeds.rb
/Users/anastasiaalt/.rbenv/versions/2.2.3/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:121:in `require': cannot load such file -- ./models/author (LoadError)

So I removed all the requires of the model in the seeds folder, tried to run the seed file again, and got the same error:

anastasiaalt ~/ThemeParties/db on master[!]
$ ruby seeds.rb
seeds.rb:3:in `<main>': uninitialized constant Author (NameError)
anastasiaalt ~/ThemeParties/db on master[!]
$
